vtkext/private/module/vtkF3DImguiConsole.h

```diff
@@ -0,0 +1,59 @@
+/**
+ * @class   vtkF3DImguiConsole
+ * @brief   An ImGui console window
+ *
+ * This class is used instead of vtkF3DConsoleOutputWindow if F3D_MODULE_UI is enabled
+ * On top of the regular behavior of printing the log in the console, all the logs are also added
+ * in an imgui window so the user can access it easily by enabling the `ui.console` option.
```

Comment:
Lets document the class, not the behavior of libf3d 

```suggestion
 * in an imgui window so the user can access it easily by call ShowConsole()
```